# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium/sandbox/linux/system_headers/linux_ucontext.h
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 2
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/chromium/sandbox/linux/system_headers/linux_ucontext.h
--- a/security/sandbox/chromium/sandbox/linux/system_headers/linux_ucontext.h	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/chromium/sandbox/linux/system_headers/linux_ucontext.h	Wed Jun 12 11:10:48 2019 +0100
@@ -5,24 +5,18 @@
 #ifndef SANDBOX_LINUX_SYSTEM_HEADERS_LINUX_UCONTEXT_H_
 #define SANDBOX_LINUX_SYSTEM_HEADERS_LINUX_UCONTEXT_H_
 
-#if defined(__ANDROID__) || defined(__native_client_nonsfi__)
+#if defined(__native_client_nonsfi__)
 
 #if defined(__arm__)
 #include "sandbox/linux/system_headers/arm_linux_ucontext.h"
 #elif defined(__i386__)
 #include "sandbox/linux/system_headers/i386_linux_ucontext.h"
-#elif defined(__x86_64__)
-#include "sandbox/linux/system_headers/x86_64_linux_ucontext.h"
-#elif defined(__mips__)
-#include "sandbox/linux/system_headers/mips_linux_ucontext.h"
-#elif defined(__aarch64__)
-#include "sandbox/linux/system_headers/arm64_linux_ucontext.h"
 #else
-#error "No support for your architecture in Android or PNaCl header"
+#error "No support for your architecture in PNaCl header"
 #endif
 
-#else  // defined(__ANDROID__) || defined(__native_client_nonsfi__)
-#error "The header file included on non Android and non PNaCl."
-#endif  // defined(__ANDROID__) || defined(__native_client_nonsfi__)
+#else  // defined(__native_client_nonsfi__)
+#error "The header file included on non PNaCl."
+#endif  // defined(__native_client_nonsfi__)
 
 #endif  // SANDBOX_LINUX_SYSTEM_HEADERS_LINUX_UCONTEXT_H_